src/consensus/params.h

```diff
@@ -25,6 +37,10 @@ struct Params {
     int64_t nPowTargetSpacing;
     int64_t nPowTargetTimespan;
     int64_t DifficultyAdjustmentInterval() const { return nPowTargetTimespan / nPowTargetSpacing; }
+    uint64_t MaxBlockSize(uint64_t nCurrentTime) const { return blockParams->MaxBlockSize(nCurrentTime); }
```

Comment:
Point taken. That's a bit too simplistic though, since we're going to have to provide parameters. I'll play around with it.